README.md

```diff
@@ -1,24 +1,89 @@
-# README
+# Reserve Classes - Back-end
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> The project we built for the Final Capstone Project is based on an app to book an appointment to attend a class. We had to follow the given design of the website, but we personalize the content, i.e., instead of booking an appointment to try a motorcycle, we built an app to book an appointment to attend a class.
 
-Things you may want to cover:
+### Entity Relationship Diagram (ERD)
+![](/images/database.png)
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby on Rails
+- PostgreSQL
 
-* Configuration
+## Kanban board
+- Click [here](https://github.com/christianbiring1/reserve_classes_back-end/projects/1) to view the Kanban board
+- Click [here](https://github.com/christianbiring1/reserve_classes_back-end/issues/7) to view the Kanban board issue
 
-* Database creation
+## Front end
+> Click [here](https://github.com/christianbiring1/reserve_classes_front-end) to view the front end repository.
 
-* Database initialization
+## Live Demo
 
-* How to run the test suite
+Click [here](https://calm-mesa-77347.herokuapp.com/) to follow the API documentation
```

Comment:
I am seeing the API doc link here but not in the actual README for develop branch, which is strange. Maybe yo ucan reference what link to go to locally to see API docs.